doc/_includes/tutorials/common/creating-a-project.md

```diff
@@ -18,14 +18,20 @@ If you choose to set a project organism, click on the "Select an Organism" drop-
 
 The organism names are derived from the [NCBI taxonomy database](http://www.ncbi.nlm.nih.gov/taxonomy).
 
-To lock sample modification in the new project, check the "Lock sample modification" box. This setting will only allow the sample owner to edit any sample data.
+Clicking the "Next" buttons allows you to select samples that have been added to your cart to the project.
 
-When you've finished entering the details for your new project, click on the "Create Project" button. You will be redirected to the projects list.
+If there are no samples currently in the cart a message will be displayed:
 
-IRIDA will automatically generate a numeric project identifier for your project. The project identifier is used by external tools for uploading sequencing data to IRIDA. The project identifier can be found in the ["Project Details" panel]({{ site.baseurl }}/user/user/project/index.html#viewing-project-details), or in the [projects list]({{ site.baseurl }}/user/user/project/index.html#viewing-existing-projects).
+![Samples]({{ site.baseurl }}/images/tutorials/common/projects/create-project-no-samples.png)
+
+When samples are in the cart, only samples that you have full permissions on can be added to a new project.  
+
+![Samples]({{ site.baseurl }}/images/tutorials/common/projects/create-project-samples.png)
 
-#### Creating a project from cart
+To prevent samples from being modified or copied withinthe new project, check the "Prevent modification and copying of these samples" box. This setting will only allow the sample owner to edit any sample data.
 
-You can create a project with samples already added to the project by using the cart.  Continue creating the project details as described above, but select the checkbox **Add samples in cart to project**.  If this checkbox is enabled any samples in the cart will be automatically added to this project.  If there are any samples which you cannot add to the project, a warning will be displayed below.
+When you've finished entering the details for your new project, click on the "Create Project" button. You will be redirected to the projects list.
```

Comment:
Change `When` to `Once`

Reply:
Updated in c549e184db052d3a09971548d3daa9301f6a10e3